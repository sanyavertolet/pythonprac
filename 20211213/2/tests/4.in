[-7241, 6659, 2122, 8359, 2753, 2984, -4317, -2316, -5884, -2501, -1780, -8915, -1763, 1055, -2538, 9920, -4804, 6975, 3053, -2765, 9549, 3592, -7830, 6852, -2564, 4960, -5766, -5356, -7601, -6142, 1155, 8369, -719, 8505, -118, -1424, -9572, 2849, 6632, -3294, 2008, -7091, 3128, 2903, 243, -1750, 6308, 311]
