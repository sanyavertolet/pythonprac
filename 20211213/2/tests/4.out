[-9572, -8915, -7830, -7601, -7241, -7091, -6142, -5884, -5766, -5356, -4804, -4317, -3294, -2765, -2564, -2538, -2501, -2316, -1780, -1763, -1750, -1424, -719, -118, 243, 311, 1055, 1155, 2008, 2122, 2753, 2849, 2903, 2984, 3053, 3128, 3592, 4960, 6308, 6632, 6659, 6852, 6975, 8359, 8369, 8505, 9549, 9920]
